views/shelf.erb

```diff
@@ -1,15 +1,17 @@
-<h1>Shelf</h1>
+<h1 aria-hidden="true">Shelf</h1>
 
-<%= erb :shelf_navigation %>
+<nav aria-labeledby="shelf-heading" class="shelf-navigation-container">
```

Comment:
```suggestion
<nav aria-label="Shelf: Current loans" class="shelf-navigation-container">
```
According to ARC Toolkit,  having `aria-labeledby` on `nav` throws a `badAriaAttribute` error. When checking the [public site](https://www.lib.umich.edu/locations-and-hours/onsite-services-during-covid-19), the navigation uses `aria-label` and it does not produce an error.

Reply:
Oh, shoot, it was a misspelling. It should of been `aria-labelledby`. I like to use the ID to heading since it's a non-aria fallback and don't need to repeat the name. Thanks for noticing this was incorrect!